packages/filter/src/components/conditionSelect/ConditionSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+
+import Select from '@xcritical/select';
+import { ISelectedFilterComponent } from '../../interfaces';
+
+
+export const ConditionSelect: React.FC<ISelectedFilterComponent> = React.memo(({
+  currentFilter,
+  filterData: {
+    condition = '',
+    column = '',
+  } = {},
+  onChange,
+}) => {
+  const conditions = useMemo(() => (currentFilter
+    ? Object.keys(currentFilter.conditions).map((key) => ({
+      ...currentFilter.conditions[key],
```

Comment:
Possibly `...currentFilter.conditions[key]` this is unnecessary.

Reply:
I keep this logic to extend the filter functionality. Select options can accept additional parameters and these parameters will be passed to the event onChange. Here we adapt object for Select component